src/components/welcome/Welcome.tsx

```diff
@@ -1,56 +1,53 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
+import { useQuery } from '@apollo/client';
 import './Welcome.css';
 import { useNavigate } from 'react-router-dom';
-
-interface User {
-  id: number;
-  name: string;
-  email: string;
-}
+import { GET_USERS } from '../../graphql/query';
+import loginClient from '../../utils/apolloClient';
```

Comment:
Afim de aprendizado e para melhorar a qualidade do seu código, dá uma estudada aqui:
https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e